### PR TITLE
Use Abseil flags instead of gflags

### PR DESCRIPTION
**Pull request by vasilvv**

### CODE REVIEW

File: tools/quic_trace_to_time_sequence_gnuplot.cc [hunk 1]
```diff
@@ -20,14 +20,19 @@
...
 #include "lib/quic_trace.pb.h"
 
-DEFINE_string(sequence, "Which time sequence to plot: send, ack or loss.", "");
-DEFINE_bool(filter_old_acks,
-            true,
-            "Do not show an ack for a packet if it was already previously "
-            "acknowledged");
+ABSL_FLAG(std::string,
+          sequence,
+          "Which time sequence to plot: send, ack or loss.",
```
Comment (DavidSchinazi) [inline tools/quic_trace_to_time_sequence_gnuplot.cc:29]: I think the two strings are reversed here: the third parameter is the default value and the fourth is the help string

File: tools/quic_trace_to_time_sequence_gnuplot.cc [hunk 2]
```diff
@@ -130,10 +137,11 @@ void PrintTimeSequence(std::istream* trace_source) {
...
 
 int main(int argc, char* argv[]) {
-  gflags::ParseCommandLineFlags(&argc, &argv, true);
+  absl::ParseCommandLine(argc, argv);
 
-  if (FLAGS_sequence != "send" && FLAGS_sequence != "ack" &&
-      FLAGS_sequence != "loss") {
+  if (absl::GetFlag(FLAGS_sequence) != "send" &&
+      absl::GetFlag(FLAGS_sequence) != "ack" &&
+      absl::GetFlag(FLAGS_sequence) != "loss") {
```
Comment (DavidSchinazi) [inline tools/quic_trace_to_time_sequence_gnuplot.cc:144]: In the next line, should it say `--sequence` instead of `--type`?

**Review by DavidSchinazi — CHANGES_REQUESTED** [review]

File: tools/quic_trace_to_time_sequence_gnuplot.cc (hunk 1 above)
Comment (vasilvv) [reply tools/quic_trace_to_time_sequence_gnuplot.cc]: Fixed.

File: tools/quic_trace_to_time_sequence_gnuplot.cc (hunk 2 above)
Comment (vasilvv) [reply tools/quic_trace_to_time_sequence_gnuplot.cc:144]: Fixed.

**Review by DavidSchinazi — APPROVED** [review]
Thanks!

**Pull request merged**